spec/spec_helper_acceptance.rb

```diff
@@ -0,0 +1,13 @@
+require 'beaker-rspec'
+require 'beaker/module_install_helper'
+require 'beaker/puppet_install_helper'
+
+RSpec.configure do |c|
+  c.before :suite do
+    unless ENV['BEAKER_provision'] == 'no'
+      run_puppet_install_helper
+      install_module_on(hosts_as('default'))
```

Comment:
It looks like `default` works because we specify that role in ci-job-configs. I'm thinking the best thing is to just install it on all `hosts`. In the future we could drop the `default` role from ci-job-configs.

Also the `install_module_on` is necessary when running on puppet-agent versions that don't have the module vendored, for example if testing against puppet5 or if the module hasn't yet been vendored into puppet-agent (during the extraction process), or never will be vendored (like mailalias).

Also we want the pipeline to install the version of the module that has new commits. I also verified that the `install_module_on` installs the module into `/etc/puppet/code/modules` which has greater precedence than the vendored module. So we end up testing the installed module instead of the vendored one.